addons/definition.json

```diff
@@ -0,0 +1,4 @@
+{
+  "game": "game/dota_addons/{ADDON_NAME}/",
+  "content": "content/dota_addons/{ADDON_NAME}/"
+}
```

Comment:
Not sure where this came from, but it shouldn't be in the folder that Git scans.